dev-env/windows/manifests/msys2.json

```diff
@@ -1,108 +1,90 @@
-        "32a70a7b95cb09bce989e326b092155f0ae52cef1529a387891da42806e32bc5",
-        "8215d34b82db62e95df47a46233d8e836902c1acb70308f1e200aa5f1ff40d7e",
-        "b97d3193f7d5ea882d19a0e005c2b202dc3acedde062fd2cc5a607484909aa49",
-        "3169f2a1662b9441a3d4943d94649334dc488d5dab75c4e10ab2d5a68e81279f"
-    ],
-    "env_add_path": ".",
-    "installer": {
-        "script": [
-            "mv $dir\\msys64\\* $dir",
-            "rm -r -Force $dir\\msys64",
-
-            "$ErrorActionPreference = 'Continue'",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman --version'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -S --noconfirm unzip zip mingw-w64-x86_64-gcc'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /perl.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /autoconf.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /autoconf-wrapper.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /automake.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /automake-wrapper.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /diffutils.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /git.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /make.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /ncurses.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /ncurses-devel.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /gzip.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /tar.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /xz.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /jq.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /netcat.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /patch.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /openssl.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /python.msys2'\"",
-            "iex \"$dir\\usr\\bin\\bash.exe -lc 'pacman -U --noconfirm /pgsql.msys2'\"",
-            "$ErrorActionPreference = 'Stop'",
-            "rm $dir\\*.msys2",
-
-            "cp \"$dir\\mingw64\\bin\\*.dll\" \"$dir\\usr\\bin\\\""
-        ]
-    },
-    "uninstaller": {
-        "script": [
-            "rm -r -Force $dir"
-        ]
+    "license": "GPL-2.0-only|BSD-3-Clause",
+    "notes": "Please run 'msys2' now for the MSYS2 setup to complete!",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/msys2/msys2-installer/releases/download/2022-01-28/msys2-base-x86_64-20220128.tar.xz",
```

Comment:
why can we drop all the other packages?

Reply:
So I have an answer to that: we can't. Scoop now works but we fail at the very first Bazel task on a missing `patch` executable.

Reply:
Thank you Gary, for sorting this out -- I did not really understand the purpose of the entries in the manifest file and oversimplified way too much.